content/1.7/back-office/shop-parameters/contact/stores/add-edit stores.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Add / Edit Stores
+weight: 1
+---
+
+# Add / Edit Store
+
+While adding or editing a store, the following fields/toggles/drop downs/parameters are displayed:
+
+1) Name
+
+This input field allows to fill the name of the store. It can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. When hovered on "name", shows the following suggestion: "Store name (e.g. City Center Mall Store). Allowed characters: letters, spaces and %s"
```

Comment:
```suggestion
This input field allows to fill the name of the store. It can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down. When the label is hovered, a tooltip is displayed: "Store name (e.g. City Center Mall Store). Allowed characters: letters, spaces and %s"
```